dictionaries/CallMap.php

```diff
@@ -375,36 +375,36 @@
 'array_fill_keys' => ['array', 'keys'=>'array', 'value'=>'mixed'],
 'array_filter' => ['associative-array', 'array'=>'array', 'callback='=>'callable(mixed,mixed=):scalar', 'mode='=>'int'],
 'array_flip' => ['associative-array<mixed,int>|associative-array<mixed,string>', 'array'=>'array'],
-'array_intersect' => ['associative-array', 'array'=>'array', 'arrays'=>'array', '...args='=>'array'],
-'array_intersect_assoc' => ['associative-array', 'array'=>'array', 'arrays'=>'array', '...args='=>'array'],
-'array_intersect_key' => ['associative-array', 'array'=>'array', 'arrays'=>'array', '...args='=>'array'],
+'array_intersect' => ['associative-array', 'array'=>'array', '...arrays='=>'array'],
+'array_intersect_assoc' => ['associative-array', 'array'=>'array', '...arrays='=>'array'],
+'array_intersect_key' => ['associative-array', 'array'=>'array', '...arrays='=>'array'],
 'array_intersect_uassoc' => ['associative-array', 'array'=>'array', 'rest'=>'array', 'key_compare_func'=>'callable(mixed,mixed):int'],
 'array_intersect_uassoc\'1' => ['associative-array', 'array'=>'array', 'rest'=>'array', 'arr3'=>'array', 'arg4'=>'array|callable(mixed,mixed):int', '...rest'=>'array|callable(mixed,mixed):int'],
 'array_intersect_ukey' => ['associative-array', 'array'=>'array', 'rest'=>'array', 'key_compare_func'=>'callable(mixed,mixed):int'],
 'array_intersect_ukey\'1' => ['associative-array', 'array'=>'array', 'rest'=>'array', 'arr3'=>'array', 'arg4'=>'array|callable(mixed,mixed):int', '...rest'=>'array|callable(mixed,mixed):int'],
 'array_is_list' => ['bool', 'array'=>'array'],
-'array_key_exists' => ['bool', 'key'=>'string|int', 'array'=>'array|ArrayObject'],
+'array_key_exists' => ['bool', 'key'=>'string|int', 'array'=>'array'],
```

Comment:
Looks like this needs added to the 7.3 diff, it was deprecated and should be removed for 7.4: https://3v4l.org/6YObs